dnsop-structured-dns-error-page.txt

```diff
@@ -459,9 +459,9 @@ Internet-Draft            Data for Filtered DNS              August 2022
 
 5.  Interoperation with RPZ Servers
 
-   This section discusses operation with an RPZ server
-   [I-D.ietf-dnsop-dns-rpz] that indicates filtering with a NXDOMAIN
-   response with the Recursion Available bit cleared (RA=0).
+   This section discusses operation with an RPZ server [RPZ] that
+   indicates filtering with a NXDOMAIN response with the Recursion
+   Available bit cleared (RA=0).
 
    When the DNS client supports this specification but the server does
    not, the server will continue replying when a query is RPZ filtered
```

Comment:
This line is not saying anything about the upgraded DNS client accepting the RPZ filtered response. We may want to say something like:
An DNS client upgraded to support this specification will continue to accept responses with NXDOMAIN and RA=0 from the RPZ server that does not support this specification.